packages/firebase_performance/firebase_performance/example/README.md

```diff
@@ -1,8 +1,54 @@
 # firebase_performance_example
 
-Demonstrates how to use the firebase_performance plugin.
+A comprehensive demo of firebase_performance plugin API usage.
 
 ## Getting Started
 
-For help getting started with Flutter, view our online
-[documentation](https://flutter.io/).
+You can build and run the app directly on emulators. To view the performance data in your own Firebase Performance console, follow the instructions below.
+
+### Android
+
+1. Follow the [instructions](https://firebase.google.com/docs/android/setup#create-firebase-project) to create your Firebase project and register an Android app.
+
+1. Download `google-services.json` and replace the existing one in `android/app/` with yours. 
+
+1. Gradle files already have the correct Firebase configuration so no need to change them. Remove the exisitng build files (`build/`) so that your own `google-services.json` will not be overriden,  and then run the app on an Android emulator.
```

Comment:
```suggestion
1. Gradle files already have the correct Firebase configuration so no need to change them. Remove the existing build files (`build/`) so that your own `google-services.json` will not be overriden,  and then run the app on an Android emulator.
```